invenio_communities/config.py

```diff
@@ -36,3 +38,18 @@
 """Possible communities sorting options."""
 
 COMMUNITIES_DEFAULT_SORTING_OPTION = 'ranking'
+
+COMMUNITIES_OAI_NAMESPACE = 'user'
+"""OAISet 'spec' namespace for community."""
+
+COMMUNITIES_OAI_FORMAT = '{namespace}-{community_id}'
```

Comment:
IMHO `COMMUNITIES_OAI_FORMAT = 'user-{community_id}`' would do the job (we don't need `COMMUNITIES_OAI_NAMESPACE`)